plutus-scb/app/PSGenerator.hs

```diff
@@ -108,7 +108,7 @@ myTypes =
     , (equal <*> (genericShow <*> mkSumType)) (Proxy @WalletEvent)
 
     -- Contract request / response types
-    , (equal <*> (genericShow <*> mkSumType)) (Proxy @ActiveEndpoint)
+    , (order <*> (genericShow <*> mkSumType)) (Proxy @ActiveEndpoint)
```

Comment:
This doesn't work because there's no `Ord ActiveEndpoint`, because there's no `Ord Aeson.Value`...

Reply:
Ah, you beat me to the change there. :-P

Okay, I'll fix it up. :-)